src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -311,6 +311,8 @@ internal bool IsDataClassificationEnabled
         /// </summary>
         internal int DataClassificationVersion { get; set; }
 
+        internal bool IsJsonSupportEnabled = false;
```

Comment:
Is it possible to expose this property on `SqlInternalConnectionTds` instead of TdsParser?
I understand that some of the other features are being tracked on the parser. But it would be good to track this on the connection instead.

Reply:
Fixed